addon/templates/components/bootstrap-inputs/-password.hbs

```diff
@@ -5,51 +5,37 @@
   label=label
   as |control|
 }}
-  {{#if hasBlock}}
-    {{yield
-      (hash
-        label=(component control.label)
-        help=(component control.help)
-        control=control
-        input=(component 'bootstrap/-input'
-          value=value
-          elementId=control.inputId
-          type='password'
-          disabled=(readonly _disabled)
-          autocomplete=autocomplete
-          maxlength=maxlength
-          minlength=minlength
-          pattern=pattern
-          placeholder=placeholder
-          readonly=readonly
-          required=required
-          srOnly=srOnly
-          tabindex=tabindex
-          change=onChange
+  {{#let (component 'bootstrap/-input'
```

Comment:
Hadn't seen this before, but `{{#let ...}}` seems super handy. Nice!

Reply:
Yup, pretty nice! Been available since ember 3.2, and has a polyfill for further backwards compatibility.